cards/cards.css

```diff
@@ -0,0 +1,160 @@
+:root{
+    --bkg-clr:linear-gradient(45deg,aqua  0%, rgba(182, 241, 252, 1) 50%, rgba(252, 174, 241, 1) 70%);
+    --sec-text-clr:gray;
+    --pri-text-clr:black;
+    --danger-clr:red;
+}
+
+img {
+    max-inline-size: 100%;
+    block-size: auto;
+    border-radius: 1rem 1rem 0rem 0rem;
+  }
+  
+  img:hover {
+    cursor: pointer;
+  }
+  
+  .card {
+    background: var(--bkg-clr);
+    display: flex;
+    flex-wrap: wrap;
+    max-width: 350px;
+    height: min-content;
+    border-radius: 1rem;
+  }
+  
+  .card:hover {
+    transition: 0.5sec;
+    box-shadow: 2px 2px 2px 2px;
+  }
+  
+  .text {
+    padding: 1.25rem 1rem 1rem 1rem;
+    flex: 1 1 100%;
+    color: var(--sec-text-clr);
+  }
+  
+  .description {
+    padding: 0.35rem 1rem 0.35rem 1rem;
+    flex: 1 1 100%;
+    color: var(--sec-text-clr);
+  }
+  
+  .titleText {
+    font-size: 1.25rem;
+    color: var(--pri-text-clr);
+    font-weight: bolder;
+  }
+  
+  .authorText {
+    font-weight: bold;
+    padding: 0.25rem 0rem 0rem 0rem;
+  }
+  
+  .foot {
+    display: flex;
+    flex-wrap: wrap;
+    flex: 1 1 100%;
+    justify-content: space-around;
+    padding: 0rem 0.75rem 0.5rem 0.75rem;
+  }
+  
+  .foot>.material-icons{
+    padding: 0.75rem;
+    border-radius: 50%;
+  }
+  
+  .card-body:hover{
+    background-color: lightgray;
+    cursor: pointer;
```

Comment:
You can give cursor pointer to the whole card as if we hover on footer the cursor is changing.

Reply:
that was intended. I tried to replicate this: [https://material-components.github.io/material-components-web-catalog/#/component/card?type=basic ](url)